### ISSUE TITLE
Missing dependency for colorlover

### ISSUE THREAD

**Issue by h4ste** [yohann84L/plot_metric#1]
Hi,

when trying to create a BinaryClassification object, I get the following error:

```python
/slurm_storage/goodwintrr/conda/envs/jamia-cantrip/lib/python3.7/site-packages/plot_metric/functions.py in <module>
      7 from sklearn.metrics import confusion_matrix, precision_recall_curve, auc, roc_curve, average_precision_score
      8 from itertools import product, cycle
----> 9 import colorlover as cl
     10 import random
     11 from statistics import mean

ModuleNotFoundError: No module named 'colorlover'
```

It looks like plot_metric depends on the package colorlover, but does not require colorlover as a dependency.

**Comment by yohann84L** [comment]
Thanks for the issue @h4ste , I'll update the package as soon as possible.

**Comment by yohann84L** [comment]
Fixed in [plot_metric 0.0.6](https://github.com/yohann84L/plot_metric/releases/tag/0.0.6).

Install with: `pip install plot_metric==0.0.6`

**Comment by huseyin01** [comment]
> pip install plot_metric==0.0.6

didn't work
